tests/framework/PostProcessors/TSACharacterizer/tests

```diff
@@ -6,6 +6,7 @@
       type = OrderedCSV
       output = 'Basic/chz.csv'
       windows_gold = 'Basic/windowsChz.csv'
+      mac_gold = 'Basic/windowsChz.csv'
```

Comment:
so windows and mac agree but linux differs now?

Reply:
Yeah, seems that the difference is just a phase of one of the Fourier modes being pi on some OSes and -pi on others. Do you want me to change that file name now that that gold file is being used for the Mac test, too?

Reply:
Nope, I'm fine with this.